lib/Nekland/YoutubeApi/Http/Auth/ServiceAuth.php

```diff
@@ -109,7 +125,7 @@ private function getPrivateKey()
      * @return mixed
      * @throws \Nekland\YoutubeApi\Exception\MissingOptionException
      */
-    private function getOption($option, $default=null)
+    protected function getOption($option, $default=null)
```

Comment:
Isn't redefined in subclass, the protected is not needed IMHO

Reply:
It is used !

Reply:
Only [used](https://github.com/Nekland/YoutubeApi/pull/13/files#diff-e3e73917769c2433b3b97c9841b84792R58), not redefined

Reply:
it still needs to be protected :)

Reply:
My bad 👍